src/Microsoft.Sbom.Api/Workflows/SbomConsolidationWorkflow.cs

```diff
@@ -90,7 +96,24 @@ public virtual async Task<bool> RunAsync()
             logger.Information($"Running consolidation on the following SBOMs:\n{string.Join('\n', consolidationSources.Select(s => s.SbomConfig.ManifestJsonFilePath))}");
         }
 
-        return await ValidateSourceSbomsAsync(consolidationSources) && await GenerateConsolidatedSbom(consolidationSources);
+        workingDir = fileSystemUtils.CreateTempSubDirectory();
+
+        try
+        {
+            return await ValidateSourceSbomsAsync(consolidationSources) && await GenerateConsolidatedSbom(consolidationSources);
+        }
+#if DEBUG
+        catch (Exception)
+        {
+            // This is here to help debug issues during active development. It will be removed before release.
+            System.Diagnostics.Debugger.Break();
+            throw;
+        }
+#endif
+        finally
+        {
+            fileSystemUtils.DeleteDir(workingDir, true);
```

Comment:
Are we sure we want to delete all of the validation output files? I'm of two minds about this- I can imagine scenarios where users may want these files around to dive into validation errors, but we obviously don't want to leave a mess behind. At a minimum, if we do decide to delete this dir, we should probably [remove the log messages that print the path](https://github.com/microsoft/sbom-tool/pull/1126/files#diff-ec5e66cabef4dcfd4b4e4183e932d33dcc18a4daef09b6ada571cf0c12912202R148), since that could get confusing to users.